Module4_Labs/Lab1_BFS_Maze/Cards/11.md

```diff
@@ -4,4 +4,46 @@
 
 <!--badges={Python:15}-->
 
-If we look at how the maze file is read, any maze is represented as a 2D array of characters. We can simply loop through this 2D array of characters until the letter 's' (start coordinates) is found. We can then return that tuple of start coordinates wherever our 's' is.
\ No newline at end of file
+In order to access the maze from the maze file, you are given the following function:
```

Comment:
sorry i forgot to include this in the original issue, but all medium and easy cards should be split into bite sized steps. please do that.

good explanation!